app.rb

```diff
@@ -63,5 +67,17 @@ class GrootMerchService < Sinatra::Base
     disable :unsecure
   end
 
+  configure do
+    scheduler = Rufus::Scheduler.new
+    logger = Logger.new STDOUT
+    logger.level = Logger::DEBUG
+    logger.datetime_format = '%Y-%m-%d %H:%M:%S '
+    jobs = Jobs.new
+    logger.info jobs.restock
+    scheduler.every '3s' do
```

Comment:
Every 3 seconds seems a bit too frequent, why don't we do every 8 hours or something?

@rauhul was also suggesting that we just run this on every vend, not sure which approach we want to go with.

Reply:
Running it on every vend might be better since I don't know how great rufus-scheduler is. As far as 3 seconds that was also for debugging. I was thinking of updating the time to every night but 8 hours would be fine if we decide to use a rufus-scheduler.

Reply:
I think the better solution is to use a scheduler so that we can make `vending` as fast as possible 

`rufus-scheduler` has a good reputation that works for in-memory scheduling, so I'm not worried about that.